src/info_dump/main.cpp

```diff
@@ -974,7 +974,7 @@ void get_vtables()
         {
             if (it.id)
             {
-                const size_t record = (size_t) * (offsets.begin() + idx);
+                const size_t record = (size_t)*(offsets.begin() + idx);
```

Comment:
This it due to some Visual Studio Community update. But actually the current clang format feels wrong for this, makes it tool like multiplication.

And according to our own clang format rules the de-refrence should not have a space between the pointer and `*`, Same for the C-style casting